Botbases/RSBot.Default/Bundle/Buff/BuffBundle.cs

```diff
@@ -16,7 +20,7 @@ public void Invoke()
                 if (Game.Player.State.LifeState != LifeState.Alive || Game.Player.HasActiveVehicle)
                     break;
 
-                var buff = SkillManager.Buffs.Find(p => !Game.Player.State.HasActiveBuff(p, out _) && p.CanBeCasted);
+                var buff = SkillManager.Buffs.Find(p => !Game.Player.State.HasActiveBuff(p, out _));
                 if (buff == null)
```

Comment:
If the buff cant be casting why it would be result?

Reply:
This was a temporary fix, I will revert now. In some cases "CanBeCasted" returns false eventhough the skill can be casted just fine. If one of us does have the time this should be checked but this PR is the wrong place anyway.

Reply:
CanbeCasted sometimes bugging because of timing. I can give #223 issue example for this. I will work on that.